src/utils.py

```diff
@@ -79,3 +80,56 @@ def chain_id(self) -> int:
         Aligned with https://chainlist.org/
         """
         return {Network.MAINNET: 1, Network.GNOSIS: 100}[self]
+
+
+class LoweredString(fields.String):
```

Comment:
I think we could take this one step further and try to ensure these Addresses are proper EthereumAddresses... Thing is that Web3 has a ChecksumAddress type.... do we think that Marshmallow can handle foreign types? Or could we use this class as a wrapper class to enforce the same criteria?

Reply:
Yeah I think so. It's definitely doable with something like this:

```python
class EthereumAddress(fields.Field):
    """Field that serializes to a string of numbers and deserializes
    to a list of numbers.
    """

    def _serialize(self, value, attr, obj, **kwargs):
        if value is None:
            return ""
        return f"{value}"

    def _deserialize(self, value, attr, data, **kwargs):
        try:
            return Address(value)
        except ValueError as error:
            raise ValidationError("Not a valid address") from error
```